### YamlMime:ManagedReference
items:
- uid: System.Runtime.Remoting.ServerException
  id: ServerException
  children:
  - System.Runtime.Remoting.ServerException.#ctor
  - System.Runtime.Remoting.ServerException.#ctor(System.String)
  - System.Runtime.Remoting.ServerException.#ctor(System.String,System.Exception)
  langs:
  - csharp
  name: ServerException
  nameWithType: ServerException
  fullName: System.Runtime.Remoting.ServerException
  type: Class
  summary: The exception that is thrown to communicate errors to the client when the client connects to non-.NET Framework applications that cannot throw exceptions.
  remarks: "ServerException uses the HRESULT COR_E_SERVER, which has the value 0x8013150E.  \n  \n ServerException uses the default <xref:System.Object.Equals%2A> implementation, which supports reference equality.  \n  \n For a list of initial property values for an instance of ServerException, see the ServerException constructors."
  syntax:
    content: >-
      [System.Runtime.InteropServices.ComVisible(true)]

      public class ServerException : SystemException
  inheritance:
  - System.Object
  - System.Exception
  - System.SystemException
  implements: []
  inheritedMembers:
  - System.Exception.Data
  - System.Exception.GetBaseException
  - System.Exception.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  - System.Exception.GetType
  - System.Exception.HelpLink
  - System.Exception.HResult
  - System.Exception.InnerException
  - System.Exception.Message
  - System.Exception.SerializeObjectState
  - System.Exception.Source
  - System.Exception.StackTrace
  - System.Exception.TargetSite
  - System.Exception.ToString
  - System.Object.Equals(System.Object)
  - System.Object.Equals(System.Object,System.Object)
  - System.Object.GetHashCode
  - System.Object.MemberwiseClone
  - System.Object.ReferenceEquals(System.Object,System.Object)
  platform:
  - net-11
  - net-20
  - netcore-10
- uid: System.Runtime.Remoting.ServerException.#ctor
  id: '#ctor'
  parent: System.Runtime.Remoting.ServerException
  langs:
  - csharp
  name: ServerException()
  nameWithType: ServerException.ServerException()
  fullName: System.Runtime.Remoting.ServerException.ServerException()
  type: Constructor
  assemblies:
  - mscorlib
  namespace: System.Runtime.Remoting
  summary: Initializes a new instance of the <see cref="T:System.Runtime.Remoting.ServerException"> class with default properties.
  remarks: "<xref:System.Runtime.Remoting.ServerException> inherits from the <xref:System.SystemException> class. The following table shows initial property values for an instance of the <xref:System.Runtime.Remoting.ServerException> class:  \n  \n|Property|Value|  \n|--------------|-----------|  \n|<xref:System.Exception.InnerException%2A>|`null`|  \n|<xref:System.Exception.Message%2A>|The empty string (\"\")|"
  syntax:
    content: public ServerException ();
    parameters: []
  platform:
  - net-11
  - net-20
  - netcore-10
- uid: System.Runtime.Remoting.ServerException.#ctor(System.String)
  id: '#ctor(System.String)'
  parent: System.Runtime.Remoting.ServerException
  langs:
  - csharp
  name: ServerException(String)
  nameWithType: ServerException.ServerException(String)
  fullName: System.Runtime.Remoting.ServerException.ServerException(String)
  type: Constructor
  assemblies:
  - mscorlib
  namespace: System.Runtime.Remoting
  summary: Initializes a new instance of the <see cref="T:System.Runtime.Remoting.ServerException"> class with a specified message.
  remarks: "<xref:System.Runtime.Remoting.ServerException> inherits from the <xref:System.SystemException> class. The following table shows initial property values for an instance of the <xref:System.Runtime.Remoting.ServerException> class:  \n  \n|Property Type|Condition|  \n|-------------------|---------------|  \n|<xref:System.Exception.InnerException%2A>|`null`|  \n|<xref:System.Exception.Message%2A>|The  string|"
  syntax:
    content: public ServerException (string message);
    parameters:
    - id: message
      type: System.String
      description: The message that describes the exception
  platform:
  - net-11
  - net-20
  - netcore-10
- uid: System.Runtime.Remoting.ServerException.#ctor(System.String,System.Exception)
  id: '#ctor(System.String,System.Exception)'
  parent: System.Runtime.Remoting.ServerException
  langs:
  - csharp
  name: ServerException(String,Exception)
  nameWithType: ServerException.ServerException(String,Exception)
  fullName: System.Runtime.Remoting.ServerException.ServerException(String,Exception)
  type: Constructor
  assemblies:
  - mscorlib
  namespace: System.Runtime.Remoting
  summary: Initializes a new instance of the <see cref="T:System.Runtime.Remoting.ServerException"> class with a specified error message and a reference to the inner exception that is the cause of this exception.
  remarks: "An exception that is thrown as a direct result of a previous exception should include a reference to the previous exception in the <xref:System.Exception.InnerException%2A> property. The <xref:System.Exception.InnerException%2A> property returns the same value that is passed into the constructor, or `null` if the <xref:System.Exception.InnerException%2A> property does not supply the inner exception value to the constructor.  \n  \n The following table shows initial property values for an instance of the <xref:System.Runtime.Remoting.ServerException> class:  \n  \n|Property|Value|  \n|--------------|-----------|  \n|<xref:System.Exception.InnerException%2A>|The inner exception reference|  \n|<xref:System.Exception.Message%2A>|The error message string|"
  syntax:
    content: public ServerException (string message, Exception InnerException);
    parameters:
    - id: message
      type: System.String
      description: The error message that explains the reason for the exception.
    - id: InnerException
      type: System.Exception
      description: The exception that is the cause of the current exception. If the  parameter is not `null`, the current exception is raised in a `catch` block that handles the inner exception.
  platform:
  - net-11
  - net-20
  - netcore-10
references:
- uid: System.SystemException
  isExternal: false
  name: System.SystemException
- uid: System.Runtime.Remoting.ServerException.#ctor
  parent: System.Runtime.Remoting.ServerException
  isExternal: false
  name: ServerException()
  nameWithType: ServerException.ServerException()
  fullName: System.Runtime.Remoting.ServerException.ServerException()
- uid: System.Runtime.Remoting.ServerException.#ctor(System.String)
  parent: System.Runtime.Remoting.ServerException
  isExternal: false
  name: ServerException(String)
  nameWithType: ServerException.ServerException(String)
  fullName: System.Runtime.Remoting.ServerException.ServerException(String)
- uid: System.String
  parent: System
  isExternal: false
  name: String
  nameWithType: String
  fullName: System.String
- uid: System.Runtime.Remoting.ServerException.#ctor(System.String,System.Exception)
  parent: System.Runtime.Remoting.ServerException
  isExternal: false
  name: ServerException(String,Exception)
  nameWithType: ServerException.ServerException(String,Exception)
  fullName: System.Runtime.Remoting.ServerException.ServerException(String,Exception)
- uid: System.String
  parent: System
  isExternal: false
  name: String
  nameWithType: String
  fullName: System.String
- uid: System.Exception
  parent: System
  isExternal: false
  name: Exception
  nameWithType: Exception
  fullName: System.Exception
